src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -69,3 +91,8 @@
   right: 0;
   top: 0;
 }
+
+.darkText {
```

Comment:
`.darkText` -> `.viewInAppLabel` (to make the class name more meaningful)

Reply:
Infact I used the same initially, but then found there was a classname .lightText in TranscriptSummary.scss and thought we may not want it to look like app specific. Also I saw in one of your comments in a PR